README.md

```diff
@@ -107,13 +107,13 @@ These are the following commands available from the `kelp` binary:
 
 The `trade` command has three parameters which are:
 
-- **botConf**: _.cfg_ file with the account details, [sample file here](examples/configs/trader/sample_trader.cfg).
+- **botConf**: full path to the _.cfg_ file with the account details, [sample file here](examples/configs/trader/sample_trader.cfg).
 - **strategy**: the strategy you want to run (_sell_, _buysell_, _balanced_, _mirror_, _delete_).
-- **stratConf**: _.cfg_ file specific to your chosen strategy, [sample files here](examples/configs/trader/).
+- **stratConf**: full path to the _.cfg_ file specific to your chosen strategy, [sample files here](examples/configs/trader/).
 
 Here's an example of how to start the trading bot with the _buysell_ strategy:
 
-`kelp trade --botConf trader.cfg --strategy buysell --stratConf buysell.cfg`
+`kelp trade --botConf ./path/trader.cfg --strategy buysell --stratConf ./path/buysell.cfg`
```

Comment:
The addition of `./path/` seems intuitive. Thanks for adding.
Let's keep the rest of the sample commands similar, i.e. replace `.path/` with `./path/` in the files below